README.md

```diff
@@ -112,16 +113,37 @@ Write a conditional statement that...
 8.  Write your javascript solution below
 ```js
 // your answer here
+let num = 50;
+if(num % 3 === 0 &&  num % 5 === 0){
+    console.log("FizBuzz");
+}
+else if(num % 3 === 0){
+    console.log("Fizz");
+}
+else if(num % 5 === 0){
+    console.log("Buzz");
+}
+else{
+    console.log(num);
+}
 ```
 
 #### BONUS
 9.  Research a [loop](https://javascript.info/while-for) so that your condition runs on every number from 0 to 100
 ```js
 // your answer here
+for(let i = 0; i <= 100; i++){
+
+}
 ```
 10.  Research a [function](https://javascript.info/function-basics) so that your condition runs on every number from 0 to whatever number is passed into the function
 ```js
 // your answer here
```

Comment:
const fizzBuzz = function (number){
    for (let i = 0; i <= number; i++){
        if (i % 3 == 0 &&  i % 5 == 0 ) {
            console.log("FizzBuzz")

        } else if (i % 3 == 0) {
            console.log("Fizz")  

        } else if (i % 5 == 0){
            console.log("Buzz");
        }
    }
}